pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["mesonpep517", "wheel", "meson==0.61.2", "ninja"] # PEP 508 specifications.
```

Comment:
it's ok to leave wheel as dependency here, it will be used during `pip install` on the machine itself regardless of pypi

Reply:
okay, I added it back.